### PR TITLE
Add koch sdf

### PR DESCRIPTION
**Pull request by kfahn22**
While this  is not an exact copy of The Art of Code's version of the Koch curve,  it is essentially his code.  I have added if statements to choose different iterations.  I couldn't find a function that returned vec2(sin(angle), cos(angle)).  Please let me know if there is one that can be utilized.

### CODE REVIEW

**Comment by kfahn22** [comment]
BTW, I am playing around with the code in morphological and get this error trying to use alphaFill (which is why I added if statements to the koch curve code.)  I am using p5.js.

's' : Loop index cannot be compared with non-constant expression

**Comment by patriciogonzalezvivo** [comment]
Have you add the following line to the top of your code?
```
#define PLATFORM_WEBGL
```

There is no dynamic arrays on WebGL 1.0, that flag before importing any function, changes the code to be back compatible with WebGL1.0

File: sdf/kochSDF.glsl [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#include "../math/const.glsl"
+
+/*
+original_author: The Art of Code
```
Comment (patriciogonzalezvivo) [inline sdf/kochSDF.glsl:4]: maybe we should put the name of the person instead of his youtube channel?

Comment (kfahn22) [reply sdf/kochSDF.glsl:4]: I agree.  I used Art of Code because another file referred to the original author that way.

**Comment by kfahn22** [comment]
I followed the pattern for alphaFill to add loop-please let me know if I need to fix.

**Pull request merged**